Makefile

```
@@ -18,3 +18,12 @@ format:
 lint:
 	pdm run mypy src
 	pdm run ruff src --fix
+
+######################
+#   docker compose   #
+######################
+delta-lake:
+	docker compose -p delta -f docker-compose-spark-delta-lake.yml up -d
+
+delta-lake-clean:
+	docker compose -p delta down -v
```

```suggestion
spark-lake:
	docker compose -p spark -f docker-compose-spark.yml up -d

spark-clean:
	docker compose -p spark down -v
```

delta-lake보다 spark로 통일하면 좋을 것 같아요~

---

delta-lake와 spark 나눴습니다!